ui/src/app/components/workflows/workflow/dynamic-parts/cron-quartz-part/cron-quartz-part.component.ts

```diff
@@ -128,28 +131,17 @@ export class CronQuartzPartComponent implements OnInit {
     if (showCron[1] !== '*' && isNaN(+showCron[1])) {
```

Comment:
The if-conditions in this method still need to be much more precise.
Example: The every-hour-every expression has to look like this:
`0 0/{x} * ? * * *`, where `{x}` can be one of  5, 10, 15, 20, 25 or 30. That's how you generate it in `onMinuteSelect`. Conceptually, the above if-condition should be like this:
`
if (showCron == '0 0/5 * ? * * *' || showCron == '0 0/10 * ? * * *' || showCron == '0 0/15 * ? * * *' || showCron == '0 0/20 * ? * * *' || showCron == '0 0/25 * ? * * *' || showCron == '0 0/30 * ? * * *')
`
Of course, you can make it better readable, reusable etc. Also for the other two if conditions, think about what are all the possible cron expressions that can be generated from the "Every hour at" and "Every Day at" options. Any cron-expression which cannot be created with the user-friendly input has to be displayed as free-text